src/metabase/lib/js.cljs

```diff
@@ -40,3 +40,24 @@
   (let [query-map (pMBQL query-map)]
     (log/debugf "query map: %s" (pr-str query-map))
     (lib.query/query (metadataProvider database-id metadata) query-map)))
+
+(defn- fix-namespaced-values
+  "This converts namespaced keywords to strings as `\"foo/bar\"`.
+
+  `clj->js` supports overriding how keyword map keys get transformed, but it doesn't let you override how values are
+  handled. So this function runs first and turns them into strings.
+
+  As an example of such a value, `(get-in card [:template-tags \"some-tag\" :widget-type])` can be `:date/all-options`."
+  [x]
+  (cond
+    (keyword? x)    (if-let [ns-part (namespace x)]
```

Comment:
This could use `qualified-keyword?`

Reply:
That's true. I'll fix that in a later PR to avoid re-running CI for this.